metrictank.go

```diff
@@ -31,19 +32,19 @@ import (
 	inKafkaMdm "github.com/raintank/metrictank/in/kafkamdm"
 	"github.com/raintank/metrictank/mdata"
 	"github.com/raintank/metrictank/mdata/chunk"
-	clKafka "github.com/raintank/metrictank/mdata/clkafka"
-	clNSQ "github.com/raintank/metrictank/mdata/clnsq"
+	"github.com/raintank/metrictank/mdata/notifierKafka"
+	notifierNsq "github.com/raintank/metrictank/mdata/notifierNsq"
```

Comment:
why do we need to override the import name for notifierNsq but not for notifierKafka?